File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file092.f90
program DRB108_atomic_orig_no
    use omp_lib
    implicit none

    integer a
    a = 0

    !$omp parallel
    !$omp atomic
    a = a+1
    !$omp end atomic
    !$omp end parallel

    print 100, a
    100 format ('a =',3i8)
end program

File: data/CodeLlama/readable_format_HPEC/cpp/file092.cpp
#include <iostream>
#include <omp.h>

int main() {
    int a = 0;

    #pragma omp parallel
    {
        #pragma omp atomic
        a = a + 1;
    }

    std::cout << "a = " << a << std::endl;

    return 0;
}
